tests/coalaCITest.py

```diff
@@ -59,6 +59,7 @@ def test_show_patch(self):
                 "-f", re.escape(filename),
                 "-b", "SpaceConsistencyTestBear",
                 "--settings", "use_spaces=True")
+            self.assertIn("Line contains ", output)  # Result message is shown
```

Comment:
IMO this is sufficient, it's unlikely that there will ever be a bug that causes the result to be shown after the patch.